symphony-bdk-template/symphony-bdk-template-freemarker/src/main/java/com/symphony/bdk/template/freemarker/UrlTemplateLoader.java

```diff
@@ -0,0 +1,40 @@
+package com.symphony.bdk.template.freemarker;
+
+import freemarker.cache.URLTemplateLoader;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Internal class used when instantiating a {@link FreeMarkerTemplate} with {@link FreeMarkerEngine#newTemplateFromUrl(String)}
+ */
+public class UrlTemplateLoader extends URLTemplateLoader {
```

Comment:
I think this class should be package-protected. I don't think developers will touch this class.